### PR TITLE
Feature#6 add company branches

### LINKED ISSUE #6: Add company branches
# REQUIREMENTS
use the activeadmin 
use ruby version 2.7.2
use rails version 6.0.3.4

# DESIGN

# Acceptance Criteria
List all required acceptance criteria
- [ ] add companies and store it
- [ ] save the name of the company
- [ ] save the direction of the company

### PR DESCRIPTION
**Pull request by LDVera**
## Description: Why did you change/implement that?
the admin can add branches of the company specifically name and address 

## Testing: Help me how can I test or look at the changes?
- run rails s
- go to the localhots:3000/admin
- introduce the admin@example.com like the user
- introduce "password" like the pass of the admin user
- go to the company tab
- click the create a new company button and create one

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,2 +1,4 @@
+# frozen_string_literal: true
+
 module ApplicationHelper
 end
```
Comment (YilKanda) [inline app/helpers/application_helper.rb:4]: If you don't need it, delete it.

File: app/helpers/companies_helper.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+module CompaniesHelper
+end
```
Comment (YilKanda) [inline app/helpers/companies_helper.rb:4]: If you don't need it, delete it.

File: app/helpers/employers_helper.rb [hunk 3]
```diff
@@ -1,2 +1,4 @@
+# frozen_string_literal: true
+
 module EmployersHelper
 end
```
Comment (YilKanda) [inline app/helpers/employers_helper.rb:4]: If you don't need it, delete it.

**Review by YilKanda — COMMENTED** [review]
If you don't need the helpers, you can remove them.

**Review by LPMendoza — APPROVED** [review]
You already have setup active admin and you can now register companies. I think you have a good advance, but you should include a screenshot of the module works.

**Review by jnegrete15 — APPROVED** [review]

**Review by OchoaHub — APPROVED** [review]

**Pull request merged**